pkg/oc/cli/admin/policy/modify_roles.go

```diff
@@ -15,10 +15,12 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	rbacv1client "k8s.io/client-go/kubernetes/typed/rbac/v1"
 	"k8s.io/kubernetes/pkg/apis/rbac"
+	kcoreclient "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset/typed/core/internalversion"
```

Comment:
We are actively removing internal client references.  These should use the external clients from `k8s.io/client-go`.  See the package two lines up.

Reply:
Replaced with `k8s.io/client-go/kubernetes/typed/core/v1` in f02d0e3. Let me know if it looks incorrect.